src/pkg/metadata/utils/appinuts.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) CloudCover
```

Comment:
is this file supposed to be called "appinputs.go"? (similar for test file).

Reply:
Right it's a typo

Reply:
Fixed